src/irr/asset/CImageWriterPNG.cpp

```diff
@@ -113,8 +113,8 @@ bool CImageWriterPNG::writeAsset(io::IWriteFile* _file, const SAssetWriteParams&
 	const auto& convertedRegion = convertedImage->getRegions().begin();
 	auto convertedFormat = convertedImageParams.format;
 
-	asset::TexelBlockInfo blockInfo(convertedFormat);
-	core::vector3du32_SIMD trueExtent = blockInfo.convertTexelsToBlocks(convertedRegion->getTexelStrides());
+	assert(convertedRegion->bufferRowLength, "Detected changes in createImageDataForCommonWriting!");
+	auto trueExtent = core::vector3du32_SIMD(convertedRegion->bufferRowLength, convertedRegion->imageExtent.height, convertedRegion->imageExtent.depth);
```

Comment:
doesn't `createImageDataForCommonWriting` create regions with non-null `bufferImageHeight` ?

Reply:
uh it does https://github.com/buildaworldnet/IrrlichtBAW/pull/468/files#diff-8114b16f36382aab5966b5b39870b592R98

Reply:
same for TGA writer

Reply:
> 
> 
> uh it does https://github.com/buildaworldnet/IrrlichtBAW/pull/468/files#diff-8114b16f36382aab5966b5b39870b592R98

so use it instead of `convertedRegion->imageExtent.height`

Reply:
done